doc/integrator/routing.rst

```diff
@@ -14,7 +14,7 @@ To add this feature, you need a `OSRM backend server <https://github.com/Project
 Adding the routing interface to the template
 --------------------------------------------
 
-For a working example, check the `alternative example desktop app <https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/apps/desktop_alt/index.html>`_ of ngeo.
+For a working example, check the `alternative example desktop app <https://github.com/camptocamp/ngeo/blob/2.3/contribs/gmf/apps/desktop_alt/index.html.ejs>`_ of ngeo.
```

Comment:
The 2.3 should be done in a mako file as that:
https://github.com/camptocamp/c2cgeoportal/blob/master/doc/integrator/create_application.rst.mako#L38

Reply:
So you want me to change doc/integrator/routing.rst  to a doc/integrator/routing.rst .mako ?

And then use something like 
`https://github.com/camptocamp/ngeo/blob/<release|version>/contribs/gmf/apps/desktop_alt/index.html.ejs`?

Reply:
yes with something like:
```
https://github.com/camptocamp/ngeo/blob/${git_branch}/contribs/gmf/apps/desktop_alt/index.html.ejs
```